jvm/sargon-android/src/test/java/com/radixdlt/sargon/MnemonicWithPassphraseTest.kt

```diff
@@ -0,0 +1,65 @@
+package com.radixdlt.sargon
+
+import com.radixdlt.sargon.extensions.derivePublicKey
+import com.radixdlt.sargon.extensions.deserializeFromJsonString
+import com.radixdlt.sargon.extensions.init
+import com.radixdlt.sargon.extensions.isValidSignature
+import com.radixdlt.sargon.extensions.serializedJsonString
+import com.radixdlt.sargon.extensions.sign
+import com.radixdlt.sargon.extensions.signature
+import com.radixdlt.sargon.extensions.validate
+import com.radixdlt.sargon.samples.sample
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+
+class MnemonicWithPassphraseTest {
+
+    @Test
+    fun testJsonRoundtrip() {
+        val mnemonicWithPassphrase = MnemonicWithPassphrase(
+            mnemonic = Mnemonic.init("zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo wrong"),
+            passphrase = "passphrase"
```

Comment:
nit: maybe "super secret" or similar is less confusing a passphrase than "passphrase" (think about newcomers who might not even know what it is...).